MoltenVK/MoltenVK/GPUObjects/MVKPipeline.h

```diff
@@ -143,8 +142,8 @@ class MVKGraphicsPipeline : public MVKPipeline {
 	VkPipelineRasterizationStateCreateInfo _rasterInfo;
 	VkPipelineDepthStencilStateCreateInfo _depthStencilInfo;
 
-  MVKVector<MTLViewport> _mtlViewports;
-  MVKVector<MTLScissorRect> _mtlScissors;
+  MVKVectorInline<MTLViewport, 8> _mtlViewports;
```

Comment:
Two more to make 16.

Reply:
Thank you for adjusting the sizes of those vectors. Optimizing this for common usage cases is exactly the right method.